kolibri/plugins/learn/assets/test/views/library-page.spec.js

```diff
@@ -39,7 +39,7 @@ const CHANNEL = {
 jest.mock('../../src/composables/useChannels');
 jest.mock('../../src/composables/useCardLayoutSpan');
 jest.mock('../../src/composables/useDevices');
-jest.mock('../../src/composables/useSearch');
+jest.mock('kolibri-common/composables/useBaseSearch');
```

Comment:
Not sure why this isn't giving the same error as the topics test suite. But this one is complaining that `this.search` is not a function.

Reply:
Fixed!